### PR TITLE
Fix duplicate events

### PR DESCRIPTION
**Pull request by tyanhan**

### CODE REVIEW

File: src/main/java/seedu/address/model/schedule/EventDescription.java [hunk 1]
```diff
@@ -36,7 +36,7 @@ public String toString() {
     public boolean equals(Object other) {
         return other == this // short circuit if same object
                 || (other instanceof EventDescription // instanceof handles nulls
-                && value.equals(((EventDescription) other).value)); // state check
+                && value.toLowerCase().equals(((EventDescription) other).value.toLowerCase())); // state check
```
Comment (vishandi) [inline src/main/java/seedu/address/model/schedule/EventDescription.java:39]: can help edit the UG regarding this changes?

Comment (tyanhan) [reply src/main/java/seedu/address/model/schedule/EventDescription.java:39]: I will update in a separate PR.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -131,7 +131,7 @@ Format: `edit INDEX [n/NAME] [p/PHONE] [tg/TELEGRAM] [gh/GITHUB] [e/EMAIL] [a/AD
 * You can remove all the person’s tags by typing `t/` without
   specifying any tags after it.
 * You can also remove telegram, github, email, or address by typing its corresponding prefix without specifying anything after it.
-* You cannot edit a person's name and/or phone number such that the person's (Name, Phone Number) combination is the same as another person's (Name, Phone Number) combination in your UniGenda.
+* You cannot edit a person's `NAME` and/or `PHONE` such that there will be duplicate persons in UniGenda. Persons are considered to be duplicates if they share the same `NAME`(case-insensitive) and `PHONE_NUMBER`.
```
Comment (hanqinilnix) [inline docs/UserGuide.md:134]:
```suggestion
* You cannot edit a person's NAME and/or PHONE_NUMBER such that there are duplicated persons in UniGenda. Persons are considered to be duplicates if they share the same NAME (case-insensitive) and PHONE_NUMBER.
```

File: docs/UserGuide.md [hunk 3]
```diff
@@ -106,7 +106,7 @@ Format: `help`
 Adds a person to UniGenda without needing complete information about the person.
 
 Format: `add n/NAME p/PHONE_NUMBER [tg/TELEGRAM] [gh/GITHUB] [e/EMAIL] [a/ADDRESS] [t/TAG]...`
-* Duplicates of (Name, Phone Number) contacts will be detected; you cannot have two people with the same combination of (Name, Phone Number).
+* You cannot add duplicate persons. Persons are considered to be duplicates if they share the same `NAME`(case-insensitive) and `PHONE_NUMBER`.
```
Comment (hanqinilnix) [inline docs/UserGuide.md:109]:
```suggestion
* You cannot add duplicated persons in UniGenda. Persons are considered to be duplicates if they share the same NAME (case-insensitive) and PHONE_NUMBER.
```

**Review by hanqinilnix — COMMENTED** [review]
What about the part for events?

File: docs/UserGuide.md (hunk 3 above)
Comment (tyanhan) [reply docs/UserGuide.md:109]: We change this to non caps then?

File: docs/UserGuide.md [hunk 4]
```diff
@@ -106,7 +106,7 @@ Format: `help`
 Adds a person to UniGenda without needing complete information about the person.
 
 Format: `add n/NAME p/PHONE_NUMBER [tg/TELEGRAM] [gh/GITHUB] [e/EMAIL] [a/ADDRESS] [t/TAG]...`
-* Duplicates of (Name, Phone Number) contacts will be detected; you cannot have two people with the same combination of (Name, Phone Number).
+* You cannot add duplicate persons. Persons are considered to be duplicates if they share the same name(case-insensitive) and phone number.
```
Comment (hanqinilnix) [inline docs/UserGuide.md:109]:
```suggestion
* You cannot add duplicate persons. Persons are considered to be duplicates if they share the same name (case-insensitive) and phone number.
```

**Review by hanqinilnix — CHANGES_REQUESTED** [review]

**Comment by hanqinilnix** [comment]
Looks good.

**Pull request merged**